docs/user_guide/model_configuration.md

```diff
@@ -321,6 +321,52 @@ configuration](#minimal-model-configuration). You must still provide
 the optional portions of the model configuration by editing the
 config.pbtxt file.
 
+## Custom Model Configuration
+
+Sometimes when multiple devices running Triton instances that share one
+model repository, it is nessecery to have models configured differently
```

Comment:
```suggestion
model repository, it is necessary to have models configured differently
```

Reply:
This is what happens when working overnight.

Reply:
Fixed